src/settings/OrderApprovals/OrderApprovals.js

```diff
@@ -1,51 +1,45 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
 import { ConfigManager } from '@folio/stripes/smart-components';
 
 import {
   MODULE_ORDERS,
+  CONFIG_APPROVALS,
 } from '../../components/Utils/const';
 import { getOrderApprovalsSetting } from '../../common/utils/getOrderApprovalsSetting';
 import OrderApprovalsForm from './OrderApprovalsForm';
-import css from './OrderApprovals.css';
 
 class OrderApprovals extends Component {
-  static propTypes = {
-    label: PropTypes.object.isRequired,
-    stripes: PropTypes.object.isRequired,
-  };
-
   constructor(props) {
     super(props);
-
     this.configManager = props.stripes.connect(ConfigManager);
   }
 
-  beforeSave = (isApprovalRequired) => (
-    JSON.stringify(isApprovalRequired)
-  );
+  beforeSave = (isApprovalRequired) => JSON.stringify(isApprovalRequired);
 
   render() {
     const { label } = this.props;
 
     return (
-      <div
+      <this.configManager
         data-test-order-settings-order-approvals
-        className={css.formWrapper}
+        configName={CONFIG_APPROVALS}
+        getInitialValues={getOrderApprovalsSetting}
+        label={label}
+        moduleName={MODULE_ORDERS}
+        onBeforeSave={this.beforeSave}
       >
-        <this.configManager
-          configName="approvals"
-          getInitialValues={getOrderApprovalsSetting}
-          label={label}
-          moduleName={MODULE_ORDERS}
-          onBeforeSave={this.beforeSave}
-        >
+        <div data-test-order-settings-order-approvals>
           <OrderApprovalsForm />
-        </this.configManager>
-      </div>
+        </div>
+      </this.configManager>
     );
   }
 }
 
+OrderApprovals.propTypes = {
+  label: PropTypes.object.isRequired,
```

Comment:
I don't think `label` is an object. Please find a proper type

Reply:
This is an object, if you trust `typeof`. but if be more precise this is a react element. Do you think I need to change to `PropTypes.element`

Reply:
PropTypes.object gives you nothing in this case, it's like typescripts's `any`.

Reply:
there is a docs regarding proptypes, please check and find the proper one https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Ok I change to `PropTypes.element`

Reply:
why do you want to limit this to an element? I've mentioned a `string`, it still don't fit.

Reply:
why not node?

Reply:
I think node is correct too, `ConfigManager` is waiting for this